examples/databricks/example_databricks_workflow.py

```diff
@@ -53,19 +53,23 @@
         databricks_conn_id=DATABRICKS_CONN_ID,
         job_clusters=job_cluster_spec,
         notebook_params=[],
+        notebook_packages=[
+            {"pypi": {"package": "simplejson"}},
+        ],
     )
     with task_group:
         notebook_1 = DatabricksNotebookOperator(
             task_id="notebook_1",
             databricks_conn_id=DATABRICKS_CONN_ID,
-            notebook_path="/Users/daniel@astronomer.io/Test workflow",
+            notebook_path="/Users/pankaj.koti@astronomer.io/Notebook1",
+            notebook_packages=[{"pypi": {"package": "Faker"}}],
```

Comment:
Is this a real package? Want to be careful we don't break the example DAG and I haven't seen a pypi package with a capital before.

Reply:
Yes, it is a real package :). I have created the notebook mentioned in the path to use this package and it ran fine with the example DAG for WorkflowTaskGroup.

Here is the PyPI listing for the package: https://pypi.org/project/Faker/